aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -156,9 +156,35 @@ struct CacheKeyFused {
   float alpha;
 };
 
+struct CacheKeyWrapper : ParamsWrapper<CacheKey> {
+  CacheKeyWrapper(const cudnnBackendDescriptorType_t operation, const Tensor& y, const Tensor& x, const Tensor& w, const IntArrayRef padding, const IntArrayRef stride, const IntArrayRef dilation, int64_t groups, bool deterministic, bool allow_tf32) {
+    memset(&(this->pod), 0, sizeof(CacheKey));
```

Comment:
These are duplicate with the `memset` in `ParamWrapper() `

Reply:
Good catch, thanks!